src/main/resources/schemas/edml-1.3.0.json

```diff
@@ -22,6 +22,20 @@
       "title": "Name of the destination table in Exasol Virtual Schema",
       "description": "The name of the resulting table in Exasol."
     },
+    "destinationName": {
+      "type": "string",
+      "description": "Name of the Exasol column.\n\nDefault: name of the document property"
+    },
+    "convertAbleMappingErrorBehaviour": {
+      "description": "Behaviour to apply in case a value is not a the expected type:\n- `NULL`: use NULL instead\n- `ABORT`: abort whole query\n- `CONVERT_OR_NULL`: try to convert the value. If not possible use NULL.\n- `CONVERT_OR_ABORT`: try to convert the value. If not possible abort the whole query.",
```

Comment:
```suggestion
      "description": "Behaviour to apply in case a value is not the expected type:\n- `NULL`: use NULL instead\n- `ABORT`: abort whole query\n- `CONVERT_OR_NULL`: try to convert the value. If not possible use NULL.\n- `CONVERT_OR_ABORT`: try to convert the value. If not possible abort the whole query.",
```